deployment/ccip/cs_test_helpers.go

```diff
@@ -157,7 +159,7 @@ func DeployChainContractsConfig(t *testing.T, env cldf.Environment, chainSelecto
 				TokenPriceStalenessThreshold: 0,
 				FeeTokens: map[config.TokenSymbol]config.FeeToken{
 					"TON": {
-						Address:                    TonTokenAddr,
+						Address:                    tvm.TonTokenAddr,
```

Comment:
Not removing the TonTokenAddr at line 47 because [core](https://github.com/smartcontractkit/chainlink/blob/92a077410a542d79de56e52d9ca31f5753406699/deployment/ccip/changeset/testhelpers/messagingtest/helpers.go#L205) is still referencing it. We can remove it once we bumped core